### PR TITLE
feat: Add plan transition caveat PLUTO-680

### PR DESCRIPTION
**Pull request by nicklem**
Mentions that customers must reanalyze commits when downgrading or transitioning between Enterprise plans

### :eyes: Live preview
* https://pluto-680-doc-update-changed-plan-d--docs-codacy.netlify.app/organizations/changing-your-plan-and-billing/#changing-your-plan-and-billing
* https://pluto-680-doc-update-changed-plan-d--docs-codacy.netlify.app/faq/general/how-can-i-change-or-cancel-my-plan/#if-youre-using-codacy-cloud

### To do
- [x] Await feature release

### CODE REVIEW

File: docs/organizations/changing-your-plan-and-billing.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
...
 Each organization on Codacy has a dedicated plan and associated billing. To make changes to the plan and billing of an organization, open your organization **Settings**, page **Plan and billing**.
 
-<!--github-marketplace-start-->
+<!--plan-caveats-start-->
 !!! note
-    **If you're using GitHub Marketplace,** make changes to your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
-<!--github-marketplace-end-->
+    -   If you're using GitHub Marketplace, update your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
+
+    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they are reanalyzed.
```
Comment (nicklem) [inline docs/organizations/changing-your-plan-and-billing.md:9]: @RaquelHipolito @ajanczak 

Updated with your suggestions. Let me know, thanks!

Comment (RaquelHipolito) [reply docs/organizations/changing-your-plan-and-billing.md:9]: @nicklem should it be more detailed stating something like: reanalysed manually and/or as new commits happen.

Comment (RaquelHipolito) [reply docs/organizations/changing-your-plan-and-billing.md:9]: @nicklem should it be more detailed stating something like: reanalysed manually and/or as new commits happen.

Comment (nicklem) [reply docs/organizations/changing-your-plan-and-billing.md:9]: @RaquelHipolito

Good idea. I can link to the [relevant page](https://docs.codacy.com/faq/repositories/how-do-i-reanalyze-my-repository/).

```suggestion
    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they're [reanalyzed](../faq/repositories/how-do-i-reanalyze-my-repository.md).
```

Comment (nicklem) [reply docs/organizations/changing-your-plan-and-billing.md:9]: Upon review, I agree that your suggestion covers a use case I wasn't considering.

```suggestion
    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they are reanalyzed manually or as new commits happen.
```

File: docs/organizations/changing-your-plan-and-billing.md [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
 Each organization on Codacy has a dedicated plan and associated billing. To make changes to the plan and billing of an organization, open your organization **Settings**, page **Plan and billing**.
 
-<!--github-marketplace-start-->
+<!--plan-caveats-start-->
 !!! note
-    **If you're using GitHub Marketplace,** make changes to your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
-<!--github-marketplace-end-->
+    -   If you're using GitHub Marketplace, update your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
+
+    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they are reanalyzed manually or as new commits happen.
```
Comment (nicklem) [inline docs/organizations/changing-your-plan-and-billing.md:9]:
```suggestion
    -   If you cancel your plan or switch to a different Enterprise plan, the Codacy UI may display outdated information for some repositories until they are reanalyzed manually or as new commits happen.
```

File: docs/organizations/changing-your-plan-and-billing.md [hunk 3]
```diff
@@ -2,10 +2,12 @@
...
 Each organization on Codacy has a dedicated plan and associated billing. To make changes to the plan and billing of an organization, open your organization **Settings**, page **Plan and billing**.
 
-<!--github-marketplace-start-->
+<!--plan-caveats-start-->
 !!! note
-    **If you're using GitHub Marketplace,** make changes to your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
-<!--github-marketplace-end-->
+    -   If you're using GitHub Marketplace, update your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
+
+    -   If you cancel your subscription or switch to a different Enterprise plan, some repositories may show outdated information in the Codacy interface. To update this information, reanalyze the repositories or push new commits.
```
Comment (nicklem) [inline docs/organizations/changing-your-plan-and-billing.md:9]:
```suggestion
    -   If you cancel your subscription or switch to a different Enterprise plan, some repositories may show outdated information in the Codacy UI. To update this information, reanalyze the repositories or push new commits.
```

**Review by RaquelHipolito — APPROVED** [review]
@nicklem Nice, looks good for me.

File: docs/organizations/changing-your-plan-and-billing.md [hunk 4]
```diff
@@ -2,10 +2,12 @@
...
 Each organization on Codacy has a dedicated plan and associated billing. To make changes to the plan and billing of an organization, open your organization **Settings**, page **Plan and billing**.
 
-<!--github-marketplace-start-->
+<!--plan-caveats-start-->
 !!! note
-    **If you're using GitHub Marketplace,** make changes to your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
-<!--github-marketplace-end-->
+    -   If you're using GitHub Marketplace, update your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
+
+    -   If you cancel your subscription or switch to a different Enterprise plan, some repositories may show outdated information in the Codacy UI. To update this information, reanalyze the repositories or push new commits.
```
Comment (nicklem) [inline docs/organizations/changing-your-plan-and-billing.md:9]:
```suggestion
    -   If you cancel your plan or switch to a different Enterprise plan, some repositories may show outdated information in the Codacy UI. To update this information, reanalyze the repositories or push new commits.
```

**Pull request merged**